lib/board.rb

```diff
@@ -0,0 +1,81 @@
+
+class Board
+  attr_accessor :cells
+
+  def initialize
+    @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+    reset!
+  end
+  def cells
+    @cells
+  end
+
+  def reset!
+
+     @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
```

Comment:
We could use some Ruby magic here and use `Array.new(9, " ")` instead of explicitly setting that array.